exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/UpsertTests.kt

```diff
@@ -109,24 +108,16 @@ class UpsertTests : DatabaseTestsBase() {
 
         withTables(excludeSettings = TestDB.ALL_H2_V1, tester) { testDb ->
             val primaryKeyValues = Pair("User A", "Key A")
-            if (testDb == TestDB.ORACLE) {
```

Comment:
Oracle does not throw an error here, because for the case of autogenerated `onUpdate` part the key columns are filtered (filtered for Oracle only)